onnxruntime_extensions/_cuops.py

```diff
@@ -372,6 +372,24 @@ def get_outputs(cls):
         ]
 
 
+class StftNorm(CustomOp):
+    @classmethod
+    def get_inputs(cls):
+        return [
+            cls.io_def('pcm_wave', onnx_proto.TensorProto.FLOAT, [None]),
```

Comment:
onnx has a definition for stft here:
https://github.com/onnx/onnx/blob/main/docs/Operators.md#STFT

Can we use this signature and names and not invent new ones ?

Reply:
I don't see any benefit of why the signature has to be same. Even the signature are same, the op domain and the exporting script are still different, you can expert there is no change switch.

since ONNX already has STFT operator, it doesn't make sense to duplicate another one in ort-extensions. The reason of this op here is efficiency. Fusing with NORM operation can increase efficiency and it also has a clear physics definition magnitudes of the frequency response.

Reply:
I can run this thing in 300ms in javascript so efficiency is not the problem (reads as: if you do it in 250ms or 300ms doesn't matter if the total inference time is 10sec).
What I mean is there is a STFT in onnx defined - don't invent your own, it is hard to get such thing removed later.
You can either fix the existing STFT or temporarily use a custom op to work around it but I think it should be temporary and for that the signatures should be the same.

Reply:
ONNX STFT signature also added now, https://github.com/microsoft/onnxruntime-extensions/blob/39a32f68c7163a24581372e9be80a05478302504/test/test_audio.py#L20

but there is no end-to-end example for that now since I am not sure if there is any reliable solution to export it, or you share one if you have it?

In the long run, ONNX STFT op has to be updated or to be deprecated to be more friendly torch exporter and remove the complex data type workaround.

Reply:
yes, I can share a script.
I have trouble with StftNorm in the presents of Stft in onnx but I would have far less trouble if StftNorm 
would be log_mel_spectrum().

Reply:
> I don't see any benefit of why the signature has to be same. 

Naive question - what is the benefit of making it different? 

If it's the same (attributes and inputs/outputs match including shapes) the custom op can override the ORT implementation of an ONNX kernel to provide a different implementation. You'd also get the ONNX shape inferencing which will help with constant folding and allocation planning. Looked like StftNorm produces a different output shape though so I assume it's not a drop in replacement for ONNX STFT.

Reply:
followed up with some internal discussion.